src/__experimental__/components/select/select.spec.js

```diff
@@ -462,4 +462,35 @@ describe('Select', () => {
       assertStyleMatch({ color: '#FFFFFF' }, styleWrapper.toJSON(), { modifier: `&:hover ${StyledIcon}` });
     });
   });
+
+  describe('renders select with one option', () => {
+    it('renders correctly', () => {
+      const wrapper = mount(
+        <Select>
```

Comment:
Maybe use `shallow` here because the generated snapshot is 3k LOC

Reply:
fixed.